helm/cluster-api/files/controlplane/bases/kubeadmcontrolplanes.controlplane.cluster.x-k8s.io.yaml

```diff
@@ -0,0 +1,88 @@
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
Kustomize base file for `kubeadmcontrolplanes.controlplane.cluster.x-k8s.io` CRD.